src/eos-companion-app-integration-helper.c

```diff
@@ -486,6 +486,7 @@ list_application_infos (GCancellable  *cancellable,
           g_autofree gchar *runtime_name = NULL;
```

Comment:
This has to be changed to `runtime_spec` as part of this commit as well.

Reply:
Agreed

Reply:
It still hasn't been addressed. In the commit `runtime_spec` is used, but the variable does not exist.